docs/development/contributing.md

```diff
@@ -36,7 +36,7 @@ always passing. The unfortunate reality of software development is sometimes
 things break. As such, `main` cannot be expected to remain reliable at all
 times. We recommend using the latest stable version of Pyodide.
 
-Pyodide follows semantic versioning (http://semver.org/) - major versions for
+Pyodide follows [semantic versioning](http://semver.org/) - major versions for
```

Comment:
I wonder if we should put some caveat here about being in version `0.x.x`. In actuality, we mostly release any sort of change as `0.x.0` and on a rare occassion release minor changes that people want sooner to `0.0.x`.

Reply:
Sorry, I don't understand the meaning clearly.

Reply:
Yeah, we only follow semver because for 0.x.y each version can have breaking changes. Pretty much each Pyodide version has breaking changes. Let's not have this discussion in this PR, to avoid blocking it, there is https://github.com/pyodide/pyodide/issues/988 already.